[Flaking tests] k8s.io/kubernetes/test/integration/storageversionmigrator - TestStorageVersionMigrationWithCRD

### Which jobs are flaking?

- https://prow.k8s.io/job-history/gs/kubernetes-ci-logs/logs/ci-kubernetes-integration-arm64-master
- https://prow.k8s.io/job-history/gs/kubernetes-ci-logs/logs/ci-kubernetes-integration-master
- https://prow.k8s.io/job-history/gs/kubernetes-ci-logs/logs/ci-kubernetes-integration-ppc64le-master

### Which tests are flaking?

`k8s.io/kubernetes/test/integration/storageversionmigrator - TestStorageVersionMigrationWithCRD`

### Since when has it been flaking?

Triage tool shows the earliest failures on [2038867093587234816 3/31/2026, 12:03:18 PM](https://prow.k8s.io/view/gs/kubernetes-ci-logs/logs/ci-kubernetes-integration-arm64-master/2038867093587234816) as per [Link](https://storage.googleapis.com/k8s-triage/index.html?text=TestStorageVersionMigrationWithCRD&test=k8s.io%2Fkubernetes%2Ftest%2Fintegration%2Fstorageversionmigrator&xjob=s3)

### Testgrid link

https://testgrid.k8s.io/sig-release-master-blocking#integration-master

### Reason for failure (if possible)

```
{Failed  === RUN   TestStorageVersionMigrationWithCRD
    testserver.go:628: Resolved testserver package path to: "/home/prow/go/src/k8s.io/kubernetes/cmd/kube-apiserver/app/testing"
    testserver.go:426: runtime-config=map[storagemigration.k8s.io/v1beta1:true]
    testserver.go:427: Starting kube-apiserver on port 35157...
    testserver.go:462: Waiting for /healthz to be ok...
    storageversionmigrator_test.go:197: Failed to update a CRD: Name: testcrd, Err: context deadline exceeded
--- FAIL: TestStorageVersionMigrationWithCRD (127.79s)
} 
```

### Anything else we need to know?

We were able to reproduce this flake locally using stress tool.
Steps to reproduce:

1. Install stress tool (# go install golang.org/x/tools/cmd/stress@latest).
2. Clone the k8s repo (git clone https://github.com/kubernetes/kubernetes.git).
3. cd kubernetes
4. go test ./test/integration/storageversionmigrator/ -c -count=1
5. /root/go/bin/stress ./storageversionmigrator.test -test.run TestStorageVersionMigrationWithCRD

```
[root@cluster-node1 kubernetes]# uname -a
Linux cluster-node1 5.14.0-611.el9.x86_64 #1 SMP PREEMPT_DYNAMIC Fri Aug 22 16:46:08 UTC 2025 x86_64 x86_64 x86_64 GNU/Linux
[root@cluster-node1 kubernetes]# go test ./test/integration/storageversionmigrator/ -c -count=1
[root@cluster-node1 kubernetes]# /root/go/bin/stress ./storageversionmigrator.test -test.run TestStorageVersionMigrationWithCRD
5s: 0 runs so far, 0 failures, 8 active
10s: 0 runs so far, 0 failures, 8 active/tmp/go-stress-20260402T210105-172346454
I0402 21:01:05.748285  866851 etcd.go:66] etcd already running at http://127.0.0.1:2379
I0402 21:01:06.084705  866851 serving.go:405] Generated self-signed cert (/tmp/kubernetes-kube-apiserver269433716/apiserver.crt, /tmp/kubernetes-kube-apiserver269433716/apiserver.key)
I0402 21:01:06.084744  866851 options.go:263] external host was not specified, using 127.0.0.1
E0402 21:01:06.676740  866851 options.go:205] The manifest file is empty, ignoring.
I0402 21:01:06.683209  866851 shared_informer.go:402] "Waiting for caches to sync"
I0402 21:01:06.692489  866851 shared_informer.go:402] "Waiting for caches to sync"
I0402 21:01:06.697890  866851 plugins.go:157] Loaded 13 mutating admission controller(s) successfully in the following order: NamespaceLifecycle,LimitRanger,TaintNodesByCondition,Priority,DefaultTolerationSeconds,DefaultStorageClass,StorageObjectInUseProtection,PodGroupProtection,RuntimeClass,DefaultIngressClass,PodTopologyLabels,MutatingAdmissionPolicy,MutatingAdmissionWebhook.
I0402 21:01:06.697914  866851 plugins.go:160] Loaded 16 validating admission controller(s) successfully in the following order: LimitRanger,PodSecurity,Priority,PersistentVolumeClaimResize,RuntimeClass,CertificateApproval,CertificateSigning,ClusterTrustBundleAttest,CertificateSubjectRestriction,PodGroupWorkloadExists,NodeDeclaredFeatureValidator,JobValidation,PodResizeValidator,ValidatingAdmissionPolicy,ValidatingAdmissionWebhook,ResourceQuota.
I0402 21:01:06.698160  866851 instance.go:240] Using reconciler: lease
I0402 21:01:06.721939  866851 handler.go:304] Adding GroupVersion apiextensions.k8s.io v1 to ResourceManager
W0402 21:01:06.721978  866851 genericapiserver.go:798] Skipping API apiextensions.k8s.io/v1beta1 because it has no resources.
I0402 21:01:06.732368  866851 cidrallocator.go:198] starting ServiceCIDR Allocator Controller
I0402 21:01:06.857831  866851 handler.go:304] Adding GroupVersion  v1 to ResourceManager
I0402 21:01:06.858152  866851 apis.go:112] API group "internal.apiserver.k8s.io" is not enabled, skipping.
I040
…/tmp/go-stress-20260402T210105-71115016
I0402 21:01:05.653589  866879 etcd.go:66] etcd already running at http://127.0.0.1:2379
I0402 21:01:06.001888  866879 serving.go:405] Generated self-signed cert (/tmp/kubernetes-kube-apiserver2983029542/apiserver.crt, /tmp/kubernetes-kube-apiserver2983029542/apiserver.key)
I0402 21:01:06.001927  866879 options.go:263] external host was not specified, using 127.0.0.1
E0402 21:01:06.155310  866879 options.go:205] The manifest file is empty, ignoring.
I0402 21:01:06.161187  866879 shared_informer.go:402] "Waiting for caches to sync"
I0402 21:01:06.169787  866879 shared_informer.go:402] "Waiting for caches to sync"
I0402 21:01:06.186652  866879 plugins.go:157] Loaded 13 mutating admission controller(s) successfully in the following order: NamespaceLifecycle,LimitRanger,TaintNodesByCondition,Priority,DefaultTolerationSeconds,DefaultStorageClass,StorageObjectInUseProtection,PodGroupProtection,RuntimeClass,DefaultIngressClass,PodTopologyLabels,MutatingAdmissionPolicy,MutatingAdmissionWebhook.
I0402 21:01:06.186735  866879 plugins.go:160] Loaded 16 validating admission controller(s) successfully in the following order: LimitRanger,PodSecurity,Priority,PersistentVolumeClaimResize,RuntimeClass,CertificateApproval,CertificateSigning,ClusterTrustBundleAttest,CertificateSubjectRestriction,PodGroupWorkloadExists,NodeDeclaredFeatureValidator,JobValidation,PodResizeValidator,ValidatingAdmissionPolicy,ValidatingAdmissionWebhook,ResourceQuota.
I0402 21:01:06.187066  866879 instance.go:240] Using reconciler: lease
I0402 21:01:06.201471  866879 handler.go:304] Adding GroupVersion apiextensions.k8s.io v1 to ResourceManager
W0402 21:01:06.201502  866879 genericapiserver.go:798] Skipping API apiextensions.k8s.io/v1beta1 because it has no resources.
I0402 21:01:06.208053  866879 cidrallocator.go:198] starting ServiceCIDR Allocator Controller
I0402 21:01:06.275453  866879 handler.go:304] Adding GroupVersion  v1 to ResourceManager
I0402 21:01:06.275747  866879 apis.go:112] API group "internal.apiserver.k8s.io" is not enabled, skipping.
I0
…
```


### Relevant SIG(s)

/sig testing

--- a/test/integration/storageversionmigrator/util.go
+++ b/test/integration/storageversionmigrator/util.go
@@ -1190,17 +1190,20 @@ func (svm *svmTest) createChaos(ctx context.Context, t *testing.T) {
 
 	noFailT := ignoreFailures{} // these create and delete requests are not coordinated with the rest of the test and can fail
 
-	const workers = 10
+	const workers = 5
 	wg.Add(workers)
 	for i := range workers {
 		go func() {
 			defer wg.Done()
 
+			ticker := time.NewTicker(100 * time.Millisecond) // 10 ops/sec
+			defer ticker.Stop()
+
 			for {
 				select {
 				case <-ctx.Done():
 					return
-				default:
+				case <-ticker.C:
 				}
 
 				_ = svm.createCR(ctx, noFailT, "chaos-cr-"+strconv.Itoa(i), "v1")

